src/cudamatrix/cu-device.h

```diff
@@ -100,14 +102,15 @@ class CuDevice {
   /// will always be a multiple of n (from properties_.textureAlignment).
   /// Otherwise, return 16, which is the stride used for CPU matrices.
   int32 GetMatrixAlignment() const;
-  
+
  private:
   CuDevice();
   CuDevice(CuDevice&); // Disallow.
   CuDevice &operator=(CuDevice&);  // Disallow.
 
+
   static CuDevice global_device_;
-  
+  cublasHandle_t handle_;
```

Comment:
I am testing and looking to see if there is a speed difference (should not
be).  After that I may merge.
Dan


On Wed, Sep 2, 2015 at 8:45 PM, Yiming Wang <notifications@github.com>
wrote:

> In src/cudamatrix/cu-device.h
> <https://github.com/kaldi-asr/kaldi/pull/112#discussion_r38603228>:
>
> >    static CuDevice global_device_;
> > -
> > +  cublasHandle_t handle_;
>
> Sorry. Fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/112/files#r38603228>.
>